Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_TouchpadWalking.cs

```diff
@@ -21,14 +50,17 @@ private void Awake()
         {
             Debug.LogError("The VRTK_TouchpadWalking script requires the VRTK_PlayerPresence script to be attached to the [CameraRig]");
         }
+
+        controllerManager = GameObject.FindObjectOfType<SteamVR_ControllerManager>();
```

Comment:
The touchpad walking script is on the CameraRig so you don't need to do this.

`this.GetComponent<SteamVR_ControllerManager>()` would work here because they're on the same game object.